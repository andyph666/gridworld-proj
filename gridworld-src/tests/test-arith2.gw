int a = 4;
int b = 6;

c = a * 6 + b / 6;
d = 3 + a * 5 - b;
e = 3 * (b % a);

print(c);
print(d);
print(e);