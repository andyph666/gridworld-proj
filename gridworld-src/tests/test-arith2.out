25
17
6